File: c++/SimplePassword.cpp
#include "stdafx.h"
#include <iostream>
#include <string>

using namespace std;

class SimplePassword {

public:
	static void password()
	{
		cout << "What is the password?" << endl;

		string password;
		getline(cin, password);

		if (password.compare("myPassword")) {
			cout << "The password is correct" << endl;
		} else {
			cout << "The password is incorrect" << endl;
		}
	};
};

File: c#/SimplePassword.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QuickProblems_Sharp
{
    class SimplePassword
    {
        public static void Password()
        {
            Console.WriteLine("What is the password?");

            string password = Console.ReadLine();

            if(password.Equals("myPassword"))
            {
                Console.WriteLine("The password is correct");
            }
            else
            {
                Console.WriteLine("The password is incorrect");
            }
        }
    }
}
